datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/dashboard.py

```diff
@@ -66,16 +74,13 @@ def export(ctx, url, integration):
         abort(str(e).replace(api_key, '*' * len(api_key)).replace(app_key, '*' * len(app_key)))
 
     payload = response.json()
-    payload.setdefault('author_info', {})
-    payload['author_info']['author_name'] = 'Datadog'
-    payload.setdefault('created_by', {})
-    payload['created_by']['email'] = 'support@datadoghq.com'
-    payload['created_by']['handle'] = 'support@datadoghq.com'
-    payload['created_by']['name'] = 'Datadog'
-    payload['created_by'].pop('icon', None)
-    output = json.dumps(payload, indent=4, sort_keys=True)
-
-    file_name = payload['board_title'].strip().lower()
+    new_payload = {field: payload[field] for field in REQUIRED_FIELDS}
+    new_payload.setdefault('author_info', {})
+    new_payload['author_info']['author_name'] = author
+
+    output = json.dumps(new_payload, indent=4, sort_keys=True)
+
+    file_name = new_payload['board_title'].strip().lower()
```

Comment:
minor comment: L104 references the old payload, should we update that to `new_payload` for consistency? https://github.com/DataDog/integrations-core/pull/6787/files#diff-862281a965bb74e88a8a043e67954e0aR104

Reply:
Sure! Good catch!